packages/fxa-auth-server/test/local/senders/email.js

```diff
@@ -1714,12 +1717,11 @@ describe('email translations', () => {
         'ru',
         'language header is correct'
       );
-      assert.include(emailConfig.subject, 'Аккаунт подтверждён');
-      // Ensure the "Connect another device" action button is translated
-      assert.include(emailConfig.html, 'Подсоединить другое устройство');
+      assert.include(emailConfig.subject, 'Добавлена альтернативная эл. почта');
+      // assert.include(emailConfig.html, 'Подсоединить другое устройство');
```

Comment:
assert left over?

Reply:
Nah, email translations depended on the strings I change and was failing test. In follow up I'll clean this up a bit.